marketplace/src/lib.rs

```diff
@@ -189,6 +231,8 @@ pub mod pallet {
 		CannotTransferMarketplaceToYourself,
 		/// NFT already listed
 		CannotListAlreadytListedNFTs,
+		/// Collection is not allowed on MP
+		CollectionNotAllowed,
```

Comment:
All new errors needs to be added to the end of the list. Otherwise you risk breaking existing apps.